scripts/stage1.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+#
+# This script is run at package build time to create the base recovery
+# environment image. This image is then modified by the postinst script when
+# the package is installed, and the recovery_sync script is used on the
+# destination system to keep it up to date with configuration changes.
+#
+# shellcheck disable=SC2001
+
+set -euxo pipefail
+
+function die() {
+	echo "$1" >&2
+	exit 1
+}
+
+function get_deps() {
```

Comment:
nit: can you mention what are the arguments, either via comments or via self-explanatory local variables like
```
local binary="$1"
local destdir="$2
```